examples/exp_of_DCRNN/dataset.py

```diff
@@ -0,0 +1,115 @@
+import os
+import zipfile
+import numpy as np
+import torch
+from torch_geometric.utils import dense_to_sparse
+from six.moves import urllib
+from signal import StaticGraphTemporalSignal
+
+
+class PemsBayDatasetLoader(object):
```

Comment:
please, consider the addition of the dataset in the `/datasets/` path, following the same interface that we already have there (a suggestion is to inherit from an already existing class (`InMemoryDataset` or `Dataset`, for instance).